cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -87,72 +87,59 @@ instance NoUnexpectedThunks Peer where
     whnfNoUnexpectedThunks _ctxt _act = return NoUnexpectedThunks
 
 
--- Node can be run in two modes.
-data ViewMode =
-    LiveView    -- Live mode with TUI
-  | SimpleView  -- Simple mode, just output text.
-
 runNode
-  :: TopologyInfo
-  -> NodeAddress
-  -> Protocol
-  -> ViewMode
-  -> LoggingLayer
-  -> TraceOptions
+  :: LoggingLayer
   -> CardanoConfiguration
   -> IO ()
-runNode topology myNodeAddress protocol viewMode loggingLayer traceOptions cc = do
+runNode loggingLayer cc = do
     let !tr = llAppendName loggingLayer "node" (llBasicTrace loggingLayer)
-    let trace'      = appendName (pack $ show $ node topology) tr
+    let trace'      = appendName (pack $ show $ node $ ccTopologyInfo cc) tr
```

Comment:
`toS` maybe from `Cardano.Prelude`?

Reply:
@ksaric  unfortunately `No instance for (StringConv NodeId Cardano.BM.Data.LogItem.LoggerName)` when using `toS`

Reply:
I meant replace `pack $ show` with `toS`.

Reply:
@ksaric I did and ghc complained with the above error.

Reply:
Ah, yes, missing `show`, my bad. Should be `toS == pack`.